contents/blog/posthog-vs-hotjar.md

```diff
@@ -0,0 +1,538 @@
+            <td>Receive notifications about new data</td>
+        </tr>
+        <tr>
+            <td><strong>PagerDuty</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Receive customized alerts from insights</td>
+        </tr>
+        <tr>
+            <td><strong>Intercom</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Extract and load data to external platforms</td>
+        </tr>
+        <tr>
+            <td><strong>Customer.io</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Sync data between platforms</td>
+        </tr>
+        <tr>
+            <td><strong>Sentry</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Ingest Sentry errors for analysis</td>
+        </tr>
+        <tr>
+            <td><strong>Segment</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Ingest events from Segment</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+> **Want more?** Visit the [PostHog the app directory](/apps) for a full list of supported integrations.
+
+## Privacy, compliance, and security
+
+Regulatory compliance can be a critical need for many teams, especially if they operate in financial or healthcare industries. Regulations such as HIPPA and GDPR can require teams to store data in certain locations, or to protect data in certain ways. 
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 400px;">
+    <thead>
+        <tr>
+            <td className="w-3/12"></td>
+            <td><strong>Hotjar</strong></td>
+            <td><strong>PostHog</strong></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>GDPR compliant</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>EU hosting or data residency available</td>
+        </tr>
+        <tr>
+            <td><strong>HIPAA compliant</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center">Self-host only</td>
```

Comment:
We don't offer self-hosting anymore, other than via Open Source deployments, which wouldn't be suitable for a BAA.